### DISCUSSION TITLE
A new ride for me

### DISCUSSION THREAD

**Discussion by chkp-sijaanh** [valkey-io/valkey#439] [Q&A] [answered]
As I was just starting to look into Redis (this past month) I found out about the licenses change.

so naturally I got redirected by the linuxfoundation.org to this fork, which I find quiet interesting.

1. Can I start contributing here? It seems that this community is here to stay.. Correct me if I'm wrong..
2. Are we going to open a discord server so I could discuss protentional issues with valkey setup/build/issues?
3. Any advice from the seniors on how to start contributing? 

 I've already made a long way in Redis' :d

**Comment by madolson** [comment, marked as answer]
The best way to contribute is ideally to fix things you have issues with, i.e. improve documentation you don't find clear or fix bugs you found. You can also take a look at issues with "help-wanted", https://github.com/valkey-io/valkey/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22. I'm trying to be a bit better than I was with Redis about highlighting tasks that people could pick up.
